govcd/vsphere_vcenter.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2023 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"net/url"
+)
+
+type VCenter struct {
+	VSphereVcenter *types.VSphereVirtualCenter
+	client         *VCDClient
+}
+
+func (vcdClient VCDClient) GetAllVcenters(queryParams url.Values) ([]*VCenter, error) {
```

Comment:
This is more of a nit, should these methods be `XxxVCenter` instead of `XxxVcenter`? To match with the type.
I know that changing this will probably affect a lot of things, not only here but in the consumer, so if you think it's too "nitpicky" just ignore it 😄

Reply:
Changed.